### DISCUSSION TITLE
all experiments stopped working with the new Chrome

### DISCUSSION THREAD

**Discussion by EceKayaMPI** [jspsych/jsPsych#2058] [Q&A] [answered]
Hi, 
After my IT-managed computer updated Chrome, I can't test my experiments on local computer or the server. The experiments currently running on the server also don't work. Should I abort all my experiments or is there a workaround for this bug? I get the jspsych error or 'the experiment failed to load'. The error I get on the console is:

[Intervention] Blocked attempt to create a WebMediaPlayer as there are too many WebMediaPlayers already in existence.

Best,
Ece

**Comment by jodeleeuw** [comment, marked as answer]
Hi Ece,

What version of jspsych are you using?

**Reply by jodeleeuw** [reply]
Also, do all of your experiments involve a lot of audio or video?

**Reply by EceKayaMPI** [reply]
I use jspsych 6.3.1 and Chrome version is 92.0.4515.107. 
The experiment loads 20 mb of sound files at the beginning of the experiment. Should I divide it to smaller chunks to be loaded separately before each block?

**Reply by jodeleeuw** [reply]
I'm not sure if that will help. It looks like Chrome has imposed a limit on the number of audio elements. How many different sound files are you loading? Can you try setting `use_webaudio` to be true in the init settings, if you aren't using it already?

**Reply by EceKayaMPI** [reply]
I have just changed use_webaudio to true and it loaded :) 
What are the implications of using web audio for the experiment? Does it affect the presentation of audio?

**Reply by jodeleeuw** [reply]
Great! 

Here's a little info on we audio: https://www.jspsych.org/overview/experiment-options/#choose-the-method-for-playing-audio-files

Basically webaudio is slightly better for timing, but a little harder to work with when you are running the experiment locally instead of on a server.

I am curious about how many audio files your experiment loaded.

**Reply by EceKayaMPI** [reply]
Oh, sorry for missing that question. The experiment loads around 410 .m4a files, 65 kb each. 
Thanks for the info and as always, thanks for solving the problem and saving the day :)
